### PR TITLE
Implements lobby view and starter Pokemon view

### PR DESCRIPTION
**Pull request by aarti156**

### CODE REVIEW

File: public/home/home.html [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-    </li>
-  <br />  
-  <h2><b>All Available Games</b></h2>
-    <li ng-repeat = "game in games track by $index">
-      <a href='#/lobby' ng-click='localStorage(game.gameId)'>{{game.gameId}}: {{game.gameName}} </a>
-    </li>
-  <br /> 
-  <a href='#/starter'>Go To StarterPokemons</a>
+    <p>Logged in as {{user.displayName}}</p>
+    {{test}}
```
Comment (hiteshlala) [inline public/home/home.html:3]: can we remove this line

File: public/lobby/lobby.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div>
+  {{ lobbytest }}
```
Comment (hiteshlala) [inline public/lobby/lobby.html]: remove

File: server/game/gameController.js [hunk 3]
```diff
@@ -12,18 +12,24 @@ module.exports = {
     var userId = req.body.userId;
     var pokemon = req.body.pokemon;
 
+    console.log("inside addPokemon function ", userId);
```
Comment (hiteshlala) [inline server/game/gameController.js:15]: remove

File: server/game/gameController.js [hunk 4]
```diff
@@ -62,6 +68,60 @@ module.exports = {
...
+        next(error);
+      });
+  },
+
+  findTurn: function (req, res, next) {
+    var gameId = req.query.gameId;
+
+    findGame({gameId: gameId})
+      .then(function (game) {
+        game.gameTurn = game.users[game.gameCounter%game.users.length].playerName;
```
Comment (hiteshlala) [inline server/game/gameController.js:117]: put spaces

**Comment by hiteshlala** [comment]
Ok - tested and good to merge

**Comment by Lindayezou** [comment]
Tested. All new and existing functionality seems to be working properly. Good job!

**Pull request merged**